ibm_catalog.json

```diff
@@ -170,7 +170,22 @@
                   ]
                 },
                 {
-                  "key": "elasticsearch_version"
+                  "key": "elasticsearch_version",
+                  "required": true,
+                  "options": [
+                    {
+                      "displayname": "v8.12",
+                      "value": "8.12"
+                    },
+                    {
+                      "displayname": "v8.10",
+                      "value": "8.10"
+                    },
+                    {
+                      "displayname": "v8.7",
```

Comment:
our module doesn't even support this version? https://github.com/terraform-ibm-modules/terraform-ibm-icd-elasticsearch/blob/b392f929de1d1cea38d1007455e138090f996ac3/variables.tf#L51-L52

I suggest to remove the `v` from the display names too - its not something ICD adds, so why would we add it

Reply:
done.